src/js_realm.hpp

```diff
@@ -689,6 +689,33 @@ bool RealmClass<T>::get_realm_config(ContextType ctx, size_t argc, const ValueTy
                 config.should_compact_on_launch_function = std::move(should_compact_on_launch_functor);
             }
 
+            static const String data_initalization_string = "dataInitialization";
```

Comment:
Is this name prior art? (ex. from the Java SDK?)

Reply:
Java calls it [`initialDataTransaction`](https://github.com/realm/realm-java/blob/master/realm/realm-library/src/main/java/io/realm/RealmConfiguration.java#L102) which actually describes it well.

Reply:
I can't say I love either of those. I see you use `initializer` in your test. I actually like that better 🤔

Reply:
If we are discussing naming, I have a few other suggestions to throw in the ring:
 `onRealmInit`, `onFirstOpen`, `initData`